front/app/routes/_main/_layout.tsx

```diff
@@ -50,6 +50,10 @@ const Menu = ({
 
   return (
     <Flex align="center" gap="4" justify="end">
+      <Text as="span" size="2">
+        ${user.balance.toLocaleString()}
```

Comment:
```suggestion
        ${new Intl.NumberFormat(undefined, { style: 'currency', currency: 'USD' }).format(
    user.balance,
  )}
```

Ajouter aussi un span en `sr-only` qui dit "Votre balance" ou quelque chose comme ça